worker/jobs/jobCleanupJob.js

```diff
@@ -21,6 +21,11 @@ module.exports = (job, done) => {
   const reqStartTime = job.data.reqStartTime;
   const dbStartTime = Date.now();
 
+  if (featureToggles.isFeatureEnabled('instrumentKue')) {
```

Comment:
why adding this here?

Reply:
This was inside the jobCleanup, as the job subject delete has the same, I thought that would be the place to be moving from jobCleanup to JobCleanupJob. 

Should I get rid off?